webpack_loader/config.py

```diff
@@ -9,12 +9,17 @@
 DEFAULT_CONFIG = {
     'DEFAULT': {
         'CACHE': not settings.DEBUG,
+        'CACHE_TTL': -1,
         'BUNDLE_DIR_NAME': 'webpack_bundles/',
         'STATS_FILE': 'webpack-stats.json',
         # FIXME: Explore usage of fsnotify
         'POLL_INTERVAL': 0.1,
         'TIMEOUT': None,
-        'IGNORE': ['.+\.hot-update.js', '.+\.map']
+        'IGNORE': ['.+\.hot-update.js', '.+\.map'],
+        'ASSETS_LOADER': {
+            'func': 'webpack_loader.utils.load_assets_from_filesystem',
+            'args': {}
```

Comment:
Technically, the func could take position and/or keyword arguments, so we'd need `'args': (), 'kwargs': {},`